driftbase/models/db.py

```diff
@@ -114,7 +112,35 @@ def check_password(self, password):
 
 tbl_user_identity = UserIdentity.__table__
 
-
+class PlayerRichPresence:
+    """
+    Rich presence information for a particular user.
```

Comment:
Please don't mix user and player terminology, they are not the same. This would be a player.

Reply:
Thanks. I will remove these doc-strings for now to avoid incorrect documentation.